gnocchi/rest/api.py

```diff
@@ -324,7 +324,7 @@ def post(self):
         enforce("create archive policy", {})
         # NOTE(jd): Initialize this one at run-time because we rely on conf
         conf = pecan.request.conf
-        valid_agg_methods = (
+        valid_agg_methods = list(
```

Comment:
how come you added list here but not in the backports?

Reply:
The backport works fine and is doing the same thing (in a different line) it seems but this one still fails for docs-gnocchi.xyz target and i'm trying to understand why